Removed references to Loki free tier (#1231)
grafana/loki: production/README.md

Currently there are five ways to try out Loki, in order from easier to hardest:

- [Grafana Cloud: Hosted Logs](#grafana-cloud-logs)
- [Run Loki locally with Docker](#run-locally-using-docker)
- [Use Helm to deploy on Kubernetes](#using-helm-to-deploy-on-kubernetes)
- [Build Loki from source](#build-and-run-from-source)
- [Get inspired by our production setup](#get-inspired-by-our-production-setup)

For the various ways to run `promtail`, the tailing agent, see our [Promtail documentation](../docs/promtail.md).

## Grafana Cloud: Hosted Logs

> Grafana is offering hosted Loki as part of our broader Grafana Cloud platform. Find instructions for getting access at [grafana.com/loki](https://grafana.com/loki).

Reviewer: ```suggestion
Grafana is offering hosted Loki as part of our broader Grafana Cloud platform. Learn more at [grafana.com/loki](https://grafana.com/oss/loki/#products-and-services).
```
